uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
+    def append_template_dir(dirname):
```

Comment:
Purely a "naming things" issue, but "append" implies that you're adding it to the end, whereas what we're actually doing is inserting at the beginning.  Inserting is obviously the correct behavior because otherwise we wouldn't be "overriding", but when I saw the name of the function my immediate thought was, "uh oh this isn't going to work" and only when I saw that the implementation didn't match the name was I reassured.